api/TextTrackCueList.json

```diff
@@ -0,0 +1,145 @@
+{
+  "api": {
+    "TextTrackCueList": {
+      "__compat": {
+        "support": {
+          "chrome": {
+            "version_added": "23"
```

Comment:
This doesn't match TextTrackCue.json, in particular the `cues` entry there which is an instance of `TextTrackCueList` and these would have to have been introduced together. Perhaps a case of the interface object not being exposed?

Reply:
It's very possible -- I confirmed the source of the data for `TextTrackCue` (#3763), and it checks out.  I'll reopen this after mdn-bcd-collector v1.1.7, which features custom tests for these APIs.

Reply:
Hmm...it seems that the results are the same for the v1.1.7 collector.

Reply:
Can you poke around manually? Was it really the case that part of the text track stuff was introduced in Chrome 9, but TextTrackCueList not until 23? The related APIs worth checking are HTMLTrackElement, HTMLMediaElement.textTracks, and TextTrack* + VTT*.

Reply:
I found the WebKit changeset which introduced the interface: https://trac.webkit.org/changeset/97926/webkit

At the time interfaces had to be explicitly exposed in DOMWindow.idl, which was done for TextTrackCue but not TextTrackCueList.

I think we can then confidently say this should be supported from the same versions as `api.TextTrackCue`, `api.HTMLTrackElement` and `api.HTMLMediaElement.textTracks`. I'll check if the data agrees.